searchcore/src/vespa/searchcore/proton/server/transactionlogmanagerbase.h

```diff
@@ -36,8 +38,7 @@ class TransactionLogManagerBase {
     StatusResult init();
 
     void internalStartReplay();
-    virtual void doLogReplayComplete(const vespalib::string &domainName,
-                                     int64_t elapsedTime) const = 0;
+    virtual void doLogReplayComplete(const vespalib::string &domainName, int64_t elapsedTime) const = 0;
```

Comment:
consider using duration for elapsed time

Reply:
Fixed